components/ArticlesList.jsx

```diff
@@ -4,14 +4,21 @@ import ArticleCard from "./ArticleCard";
 
 export default function ArticlesList() {
     const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
+    const loadingMessage = 'Articles loading...';
 
     useEffect(() => {
         getArticles().then(articles => {
             setArticles(articles)
+            setIsLoading(false)
+        })
+        .catch((error) => {
+            setIsLoading(false)
```

Comment:
Where are we returning this error to?
The convention with errors would be to have some state (isError) and conditionally render an error message if isError is true